.chloggen/add-api-integration-test.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
Thanks for following the changelog process, but we can exclude the entry in this case because it is only a test change.